src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
```

Comment:
improve react imports order

https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf